CSS/Homepage.css

```diff
@@ -23,3 +30,43 @@
     align-items: center;
     height: 80px;
 }
+
+/* question-one */
+.form__list {
+    display: flex;
+    flex-direction: column;
+    min-height: 50vh;
+    justify-content: space-between;
+}
+
+.title__question {
+    margin: 0px 35px 40px 35px;
+    ;
+}
+
+.form__item {
+    margin: 0px 35px 16px 45px;
```

Comment:
right and left margins should be done at layout level

Reply:
I already looked at the design and put the margins I had in the design